src/tests/ftest/osa/offline_parallel_test.py

```diff
@@ -12,8 +12,7 @@
 from daos_utils import DaosCommand
 from dmg_utils import check_system_query_status
 from exception_utils import CommandFailure
-from test_utils_pool import TestPool
-from test_utils_base import LabelGenerator
+from test_utils_pool import add_pool
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'